ocs_samples/basic_samples/SDS/DotNet/SdsClientLibraries/SdsClientLibraries/Program.cs

```diff
@@ -504,7 +512,7 @@ public static async Task<bool> MainAsync(bool test = false)
             }
             finally
             {
-                //step 20
+                //step 21
                 Console.WriteLine("Cleaning up");
```

Comment:
Maybe add Console.WriteLine(); in front of this line too so it separates "Cleaning Up" it from Window Data

Reply:
Space added between Window Data and "Cleaning Up"